### ISSUE TITLE
Adobe reader advanced search: NVDA does not trigger focus change in the search results tree view and does not report the results

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#9139]
### Steps to reproduce:
1. Open NVDA
2. Open a pdf document with more pages
3. Press ctrl+shift+f to open advanced search window
4. Search for a word that gives at least two results
5. In the tree view, navigate with up and down arrow

### Actual behavior:
NVDA does not read the results even though the focus moves in the tree view. When pressing tab and shift+tab, NVDA reports finally the current item focused. This is the developer info for an item in the results tree view:

```
INFO - globalCommands.GlobalCommands.script_navigatorObject_devInfo (18:38:43.049):
Developer info for navigator object:
name: None
role: ROLE_TREEVIEW
states: STATE_FOCUSABLE, STATE_FOCUSED
isFocusable: True
hasFocus: True
Python object: <NVDAObjects.IAccessible.adobeAcrobat.RootNode object at 0x06A913F0>
Python class mro: (<class 'NVDAObjects.IAccessible.adobeAcrobat.RootNode'>, <class 'NVDAObjects.IAccessible.adobeAcrobat.AcrobatNode'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: None
location: RectLTWH(left=68, top=567, width=670, height=751)
value: None
appModule: <'appModuleHandler' (appName u'acrord32', process ID 11572) at address 6bc01f0>
appModule.productName: u'Adobe Acrobat Reader DC'
appModule.productVersion: u'19.10.20069.311970'
TextInfo: <class 'NVDAObjects.NVDAObjectTextInfo'>
windowHandle: 67300
windowClassName: u'AVL_AVView'
windowControlID: 0
windowStyle: 1442840576
windowThreadID: 6944
windowText: u'AVSearchTreeViewClass'
displayText: u'C:\\U...\\braille-sense-polaris_user-manual_\nJAWS, NVDA, SuperNova, System Acc\nC:\\U...\\braille-sense-polaris_user-manual_\nJAWS, NVDA, SuperNova, System Acc'
IAccessibleObject: <POINTER(IAccessible) ptr=0x71c51b0 at 6a516c0>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=67300, objectID=-4, childID=0
IAccessible accName: None
IAccessible accRole: ROLE_SYSTEM_OUTLINE
IAccessible accState: STATE_SYSTEM_FOCUSED, STATE_SYSTEM_FOCUSABLE, STATE_SYSTEM_VALID (1048580)
IAccessible accDescription: None
IAccessible accValue: None
```

 ### Expected behavior:
NVDA should reports the search results when navigating with up and down arrow through the tree view.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2018.4

#### Windows version:
Windows 10 1809 Update

#### Name and version of other software in use when reproducing the issue:
Adobe reader 19.10.20069.311970

#### Does the issue still occur after restarting your PC?
yes

#### Have you tried any other versions of NVDA?
yes, 2017.3 on Windows 8.1.

**Comment by seanbudd** [comment]
@Adriani90 - is this still an issue with the latest versions of the software and NVDA?

**Comment by Adriani90** [comment]
Yes this is still an issue in Adobe Continuous Release I Version 2025.001.20623 | 64-Bit.

This seems to need an external fix, the first child object of the tree view containing the search results is the path and the name of the file,. The search results are starting on the second child object of the tree view. But NVDA does not read anything when arrowing through the tree view.

**Comment by SaschaCowley** [comment]
@Adriani90 can you please report this issue to Adobe and include a link to the ticket here.
